api/v1alpha1/multiclusterservice_types.go

```diff
@@ -103,6 +103,10 @@ type ServiceSpec struct {
 	DriftIgnore []libsveltosv1beta1.PatchSelector `json:"driftIgnore,omitempty"`
 	// DriftExclusions specifies specific configurations of resources to ignore for drift detection.
 	DriftExclusions []sveltosv1beta1.DriftExclusion `json:"driftExclusions,omitempty"`
+
+	// +kubebuilder:default:=false
+	// ContinueOnError specifies if the services deployment should continue if an error occurs.
```

Comment:
```suggestion
	// +kubebuilder:default:=false

	// ContinueOnError specifies if the services deployment should continue if an error occurs.
```

formatting to avoid appearing as part of the comment. Why do we need the explicit default? I recall asking a similar question, and IIRC, the answer was similar to "to ensure more clarity" (or smth), is there the same intention?

Reply:
Correct. Will reformat.